src/riak_pb_messages.csv

```diff
@@ -31,6 +31,8 @@
 30,RpbResetBucketResp,riak
 31,RpbGetBucketTypeReq,riak
 32,RpbSetBucketTypeReq,riak
+33,Rpbgetbucketkeypreflistreq,riak
```

Comment:
Make sure this is capitalized in the same way as the messages in the .proto file. Erlang lowercases everything, but Java and Python codegen rely on correct capitalization.